### ISSUE TITLE
Implement basic gravity model, e.g. as `si_gravity()`

### ISSUE THREAD

**Issue by Robinlovelace** [Robinlovelace/simodels#26]

**Comment by R-icntay** [comment]
Happy to help!

Just to run this with @Robinlovelace , @adamdennett 

This is what we would be implementing, right? The total unconstrained model?

$T_{ij} = k V_i^\mu W_j^\alpha d_{ij}^-\beta$

**Comment by Robinlovelace** [comment]
Yes, that looks like the one!

$\mu \alpha \beta$

Can be set to 1 by default I think so it would be something like...

**Comment by adamdennett** [comment]
Yeah, that will do for a basic model, although set beta to -1 rather than 1 in any initial model. K can also be calculated simply as all flows in the system / the rest of the model, but 1 as a starter is fine

**Comment by Robinlovelace** [comment]
Thanks Adam for this key input!
